### DISCUSSION TITLE
quasar responsive multi image carousel

### DISCUSSION THREAD

**Discussion by salmanshahflowingstreams** [quasarframework/quasar#13087] [General - Components / Directives / etc]
hi,

https://quasar.dev/vue-components/carousel#qcarousel-api

this one is showing carousel for multi items 
but its not showing anything to handle this for the responsive 
for example i want to show 6 on desktop and 2 on the mobile 
how can i do it 
or any other package

**Comment by ontwikkelfabriek** [comment]
@salmanshahflowingstreams 

quick and dirty:
https://codepen.io/ontwikkelfabriek/pen/xxpWbGq
